package.json

```diff
@@ -91,22 +89,20 @@
     "./dist/bson.esm.js": "./dist/bson.browser.esm.js"
   },
   "engines": {
-    "node": ">=6.9.0"
+    "node": ">=14.20.1"
```

Comment:
Current 14.x release. This is what we did when we picked the 12 ver for the driver we set it to the minor that was available at the time, we only ever pull in the latest minor for our testing so I think this is effective at communicating we test on 14.20+ (I can drop the patch version if we want)